environment.yml

```
@@ -16,15 +16,16 @@ dependencies:
   - jsonpickle
   - lmoments3
   - numba
-  - numpy >=1.16
-  - pandas >=0.23,<2.2
+  - numpy >=1.20.0
+  - pandas >=2.2.0
   - pint >=0.9
   - poppler >=0.67
+  - pyarrow  # Strongly encouraged for Pandas v2.2.0+
   - pyyaml
   - scikit-learn >=0.21.3
-  - scipy >=1.2
+  - scipy >=1.10.0
```

```suggestion
  - scipy >=1.9.0
```
The PR's top comment and CHANGES says 1.9 ?

---

Indeed.